openreview/conference/templates/programchairWebfield.js

```diff
@@ -1077,10 +1096,45 @@ var displaySortPanel = function(container, sortOptions, sortResults, searchResul
     sortResults($(container).find('.form-sort').val(), true);
     return false;
   });
-  $(container + ' .form-search').on('keyup', _.debounce(function() {
-    var searchText = $(container + ' .form-search').val().toLowerCase().trim();
-    searchResults(searchText);
-  }, 300));
+  $(container + ' .form-search').on('keyup', function (e) {
+    var searchText = $(container + ' .form-search').val().trim();
+    var searchLabel = $(container + ' .form-search').prevAll('strong:first').text();
+    if (enableQuery){
+      $('#group-container').find('button.btn.btn-export-data').text('Export')
+      conferenceStatusData.filteredNotes = null
+    }
+    $(container + ' .form-search').removeClass('invalid-value');
+  
+    if (enableQuery && searchText.startsWith('+')) {
+      // filter query mode
+      if (searchLabel === 'Search:') {
+        $(container + ' .form-search').prevAll('strong:first').text('Query:');
+        $(container + ' .form-search').prevAll('strong:first').after($('<span/>', {
+          class: 'glyphicon glyphicon-info-sign'
+        }).hover(function (e) {
+          $(e.target).tooltip({
+            title: "query format is {filter}=value<br/>\nand can be combined with AND/OR<br/>\neg. +number<50 AND reviewers=\"first last\"<br/>\navailable filters:".concat(Object.keys(propertiesAllowed).join('<br/>')),
+            html: true,
+            placement: 'bottom'
+          });
+        }));
+      }
+  
+      if (e.key === 'Enter') {
```

Comment:
This should also check for tab

Reply:
looks like we can't catch tab key up
not sure will it worth to add a keydown handler for this